Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "ammonia"
-version = "0.7.0" # remember to update html_root_url
+version = "0.7.0-rc1" # remember to update html_root_url
```

Comment:
Should this be `1.0-rc1` and in the other PR?

Reply:
Yes 🤦‍♂️

Reply:
And umm.. "remember to update html_root_url". Does docs.rs work for `-rc` releases?